cmd/kubeadm/app/cmd/token.go

```diff
@@ -116,9 +115,9 @@ func NewCmdToken(out io.Writer, errW io.Writer) *cobra.Command {
 		},
 	}
 	createCmd.Flags().DurationVar(&tokenDuration,
-		"ttl", kubeadmconstants.DefaultTokenDuration, "The duration before the token is automatically deleted (e.g. 1s, 2m, 3h). 0 means 'never expires'.")
+		"ttl", kubeadmconstants.DefaultTokenDuration, "The duration before the token is automatically deleted (e.g. 1s, 2m, 3h). If set to '0', the token will never expire.")
 	createCmd.Flags().StringSliceVar(&usages,
-		"usages", kubeadmconstants.DefaultTokenUsages, "The ways in which this token can be used. Valid options: [signing,authentication].")
+		"usages", kubeadmconstants.DefaultTokenUsages, "The ways in which this token can be used. Valid options are 'signing' and 'authentication'.")
```

Comment:
We might want to include some note about how multiple usages can be specified (with commas or by passing `---usages` multiple times, IIRC).

Reply:
this became:
```
Describes the ways in which this token can be used. You can pass --usages
multiple times or provide a comma separated list of options. Valid options
are 'signing' and 'authentication'.
```